wordleCracker.py

```diff
@@ -133,6 +219,9 @@ def narrow(wordList):
 
 def solve():
     #Find potential words given a starting word.
+    """
+    Why are you making me guess "crane" for my first turn :( 

```

Comment:
Because I saw a video by 3b1b on solving wordles with information theory, and he found that crane was one of the best words - Although, since I'm not using machine learning, perhaps it's not the best word.